aldryn_reversion/admin.py

```diff
@@ -153,3 +158,127 @@ def revision_view(self, request, object_id, version_id,
             }
             return render_to_response(self.revision_confirmation_template,
                                       context, RequestContext(request))
+
+    @transaction.atomic
+    def recover_view(self, request, version_id,
+                      extra_context=None):
```

Comment:
Why the break and the weird indentation? This would fit comfortably on a single line.